rebasehelper/cli.py

```diff
@@ -48,7 +49,7 @@ def build_parser(available_choices_only=False):
 
         group = parser.add_mutually_exclusive_group()
         current_group = 0
-        for option in traverse_options(OPTIONS):
+        for option in traverse_options(OPTIONS + plugin_runner.get_options()):
```

Comment:
I don't like the name, I think `plugin_manager` would be more fitting, as it does more than just run plugins.